src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculable {
```

Comment:
Abstract classes should not be used to set behavior for classes unless they provide some implementation that will be shared across multiple subclasses. Consider using an interface if no shared implementation is needed, or ensure that the abstract class provides a base implementation that subclasses can benefit from.